File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/13.primePairs.js
function primePairs(input) {

    let startValueToFirstPair = Number(input[0]);
    let startValueToSecondPair = Number(input[1]);
    let differenceBetweenStartAndEndValueToFirstPair = Number(input[2]);
    let differenceBetweenStartAndEndValueToSecondPair = Number(input[3]);

    let endValueToFirstPair = startValueToFirstPair + differenceBetweenStartAndEndValueToFirstPair;
    let endValueToSecondPair = startValueToSecondPair + differenceBetweenStartAndEndValueToSecondPair;

    for (let i = startValueToFirstPair; i <= endValueToFirstPair; i++) {
        for (let j = startValueToSecondPair; j <= endValueToSecondPair; j++) {
            if (i % 2 != 0 && i % 3 != 0 && i % 5 != 0 && i % 7 != 0 && j % 2 != 0 && j % 3 != 0 && j % 5 != 0 && j % 7 != 0) {
                console.log(`${i}${j}`);
            }
        }
    }
}
primePairs(['10', '30', '9', '6']);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/13.prime_pairs.py
start_value_to_first_pair = int(input())
start_value_to_second_pair = int(input())
difference_between_start_and_end_value_to_first_pair = int(input())
difference_between_start_and_end_value_to_second_pair = int(input())

endValueToFirstPair = start_value_to_first_pair + difference_between_start_and_end_value_to_first_pair
endValueToSecondPair = start_value_to_second_pair + difference_between_start_and_end_value_to_second_pair

for i in range(start_value_to_first_pair, endValueToFirstPair + 1):
    for j in range(start_value_to_second_pair, endValueToSecondPair + 1):
        if i % 2 != 0 and i % 3 != 0 and i % 5 != 0 and i % 7 != 0 and j % 2 != 0 and j % 3 != 0 and j % 5 != 0 and j % 7 != 0:
            print(f'{i}{j}')
